app/routes.js

```diff
@@ -24,21 +28,14 @@ module.exports = function(app) {
         res.sendfile('index.html');
     });
 
-    // get all screen ids
-    app.get('/api/screen_ids', function(req, res) {
+    // get fields of all screens
+    app.get('/api/screens/:field/', function(req, res) {
         Screen
             .find({})
-            .select('_id')
+            .select(req.params.field)
```

Comment:
It's a field in the context of database.. i.e. a column name. This is what they're referred to in the MongoDB documentation. I think they avoid the use of the word "key" as that usually refers to unique identifiers such as ``_id``.